src/core/server/core_app/assets/favicons/safari-pinned-tab.svg

```diff
@@ -1,45 +1 @@
-<?xml version="1.0" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
- "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
```

Comment:
Getting rid of XML headers intentional?

Reply:
Yes; An SVG document fragment can stand by itself as a self-contained file or resource, in which case the SVG document fragment is an SVG document, or it can be embedded inline as a fragment within a parent HTML or XML document. [[ref](https://svgwg.org/svg2-draft/single-page.html#struct-NewDocumentOverview)]